documentation/src/main/asciidoc/user_guide/integrations_hibernate_cache.adoc

```diff
@@ -11,14 +11,23 @@ This chapter focuses on what you need to know to use Infinispan as second-level
 Applications running in environments where Infinispan is not default cache provider for Hibernate will need to depend on the correct cache provider version.
 The Infinispan cache provider version suitable for your application depends on the Hibernate version in use:
 
-If using Hibernate 5.2 or higher, the Maven coordinates are the following:
+If using Hibernate 5.3 or higher, the Maven coordinates are the following:
```

Comment:
I guess it should say:

> If using Hibernate 5.3, the Maven coordinates are the following:

Once we release higher versions we'll have new `-vXX` projects?

Reply:
Right.

I think that `-vXX` sets the expectations more clear. I could have moved 5.2 support from `main` to `v52` packages but moving stuff around just because we have a newer version seems wrong to me. This approach should not break users who use FQCN for region factory.